UnitTests/Static_HelperTests.cs

```diff
@@ -0,0 +1,181 @@
+        }
+
+        [TestCase(15, 10, 5)]
+        [TestCase(-10, -10, 0)]
+        [TestCase(-10, 10, -20)]
+        [TestCase(0, 5, -5)]
+        [TestCase(20, 20, 0)]
+        public void Sub_WhenFirstNumAndSecondNumInitialized_ShouldFindSubstractionOfTwoNumbers(
+            double firstNum, double secondNum, double expectedResult)
+        {
+            double actualResult = Static_Helper.Sub(firstNum, secondNum);
+            Assert.AreEqual(expectedResult, actualResult);
+        }
+
+        [TestCase(3, 2, 6)]
+        [TestCase(10, 0, 0)]
+        [TestCase(-2, -4, 8)]
+        [TestCase(-5, 2, -10)]
+        [TestCase(0, 20, 0)]
+        public void Mult_WhenFirstNumAndSecondNumInitialized_ShouldFindMultiplicationOfTwoNumbers(
+            double firstNum, double secondNum, double expectedResult)
+        {
+            double actualResult = Static_Helper.Mult(firstNum, secondNum);
+            Assert.AreEqual(expectedResult, actualResult);
+        }
+
+        [TestCase(10, 2, 5)]
+        [TestCase(10, -2, -5)]
+        [TestCase(-20, -5, 4)]
+        [TestCase(5, 4, 1.25)]
+        [TestCase(-10, 8, -1.25)]
+        public void Div_WhenFirstNumAndSecondNumInitializedAndSecondNumIsNotZero_ShouldFindDivisionOfTwoNumbers(
+            double firstNum, double secondNum, double expectedResult)
+        {
+            double actualResult = Static_Helper.Div(firstNum, secondNum);
+            Assert.AreEqual(expectedResult, actualResult);
+        }
+
+        [Test]
+        public void Div_WhenFirstNumAndSecondNumInitializedAndSecondNumIsZero_ShouldThrowDivideByZeroException()
+        {
+            try
+            {
+                Static_Helper.Div(10, 0);
+            }
+            catch (DivideByZeroException ex)
+            {
+                Assert.AreEqual("Second argument was zero!", ex.Message);
+                Assert.Pass();
+            }
+            Assert.Fail();
+        }
+
+        [TestCase(0, 32)]
+        [TestCase(-8, 17.6)]
+        [TestCase(7, 44.6)]
+        [TestCase(-25, -13)]
+        [TestCase(23, 73.4)]
+        public void CelsiusToFahrenheit_WhenTemperatureCelsiusPassed_ShouldReturnTemperatureInFahrenheit(
+            double tempInCelsius, double expectedResult)
+        {
+            double actualResult = Static_Helper.CelsiusToFahrenheit(tempInCelsius);
+            Assert.AreEqual(expectedResult, actualResult);
+        }
+
+        [TestCase(0, -17.78)]
+        [TestCase(5, -15)]
+        [TestCase(7, -13.89)]
+        [TestCase(50, 10)]
+        [TestCase(59, 15)]
+        public void FahrenheitToCelsius_WhenTemperatureFahrenheitPassed_ShouldReturnTemperatureInCelsius(
+            double tempInFahrenheit, double expectedResult)
+        {
+            double actualResult = Static_Helper.FahrenheitToCelsius(tempInFahrenheit);
+            Assert.AreEqual(expectedResult, actualResult);
+        }
+
+        [TestCase("Hello", 1, 5)]
```

Comment:
It looks like `StrCount` count not from zero but from 1
Is is like in task description?